allauth/mfa/tests/test_views.py

```diff
@@ -60,6 +61,7 @@ def test_activate_totp_with_unverified_email(
     }
 
 
+@patch("allauth.account.app_settings.EMAIL_NOTIFICATIONS", True)
```

Comment:
Instead of patching, let's use the regular `pytest` facilities:

```
def test_foo(settings, ...):
    settings.ACCOUNT_EMAIL_NOTIFICATIONS = True
    ...
```

Reply:
patch statements changed in pytest style statements.